Sources/Commands/SwiftTestTool.swift

```diff
@@ -651,6 +652,7 @@ final class ParallelTestRunner {
         var unitTest: UnitTest
         var output: String
         var success: Bool
+        var time: Double
```

Comment:
we typically use DispatchTimeInterval for this purpose

Reply:
That's a good point.  @Austinpayne there is some code in https://github.com/apple/swift-package-manager/blob/5f5be413e8741aa91ba52d32580d77f5916aeb79/Sources/SPMBuildCore/PluginInvocation.swift#L432 that uses DispatchTime for plugins and then computes the interval that may be useful to look at.  There are then some functions in `DispatchTimeInterval+Extensions.swift` to convert that to other types.